File: Work/Conditions/timegreeter.py
#Ryan R, Time Greeter

import time
curent =time.time()
local_time = time.localtime(curent)
hour = local_time.tm_hour

name = input("What is your name?")
if (6 < hour < 11):
    print("Good Morning")
elif (hour < 14):
    print("Good Afternoon")
elif (hour <20):
    print("Good Night, sleep tight, don't let the bed bugs bite")
elif (hour <6 )

File: Work/Conditions/timegreeter.c
//Ryan Rowbury, Time Greeter
#include <stdio.h>
#include <time.h>
int hour;

int main(void){
time_t now = time(NULL);
    struct tm *tm_truckt = localtime(&now);
    int hour = tm_truckt->tm_hour;

    if (hour < 11){
        printf("Good morning");
    }else if(hour < 17){
        printf("Good afternoon");
    }else if (hour < 22){
        printf("Good evening");
    }else if (hour < 6){
        printf("Welcome to the AM my G bro");
    }else if (hour > 22){
        printf("Go to be lil bro");
    }else{
        printf("You are outside the boundries of time, so good luck");
    }
}